common/reviews/api/heft.api.md

```diff
@@ -0,0 +1,170 @@
+## API Report File for "@rushstack/heft"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AsyncSeriesBailHook } from 'tapable';
+import { AsyncSeriesHook } from 'tapable';
+import { CommandLineAction } from '@rushstack/ts-command-line';
+import { CommandLineFlagParameter } from '@rushstack/ts-command-line';
+import { ICommandLineActionOptions } from '@rushstack/ts-command-line';
+import { IPackageJson } from '@rushstack/node-core-library';
+import { ITerminalProvider } from '@rushstack/node-core-library';
+import { SyncHook } from 'tapable';
+import { Terminal } from '@rushstack/node-core-library';
+
+// @public (undocumented)
+export abstract class ActionHooksBase {
+    // (undocumented)
+    readonly afterLoadActionConfiguration: AsyncSeriesHook;
+    // (undocumented)
+    readonly loadActionConfiguration: AsyncSeriesHook;
+    readonly override: AsyncSeriesBailHook<IActionDataBase<ActionHooksBase>>;
+}
+
+// @public (undocumented)
+export type Build = IBuildActionData;
```

Comment:
Why are these types aliased?

Reply:
These are aliased for API simplicity in hooks. The idea is that you can say something like `compilation.hooks.build.tap('myplugin', (build: Build /* instead of IBuildActionData */) => { ... });`

Reply:
I'm open to refactoring this later.